ftplugin/markdown.vim

```diff
@@ -14,6 +14,14 @@ if !exists("g:vmt_dont_insert_fence")
     let g:vmt_dont_insert_fence = 0
 endif
 
+if !exists("g:vmt_list_item_char")
```

Comment:
I think it is better to keep the default behavior when add options. Otherwise, users will be confused by they do nothing but the Table of Contents changed.

Reply:
yeah,  I will fix it .